### ISSUE TITLE
Extend internal measure functions

### ISSUE THREAD

**Issue by be-marc** [mlr-org/mlr3automl#17]
Extent the `*_internal_measure()` functions. 

https://lightgbm.readthedocs.io/en/stable/Parameters.html#metric-parameters (`metric`)

https://xgboost.readthedocs.io/en/stable/parameter.html (`eval_metric`) 

https://catboost.ai/en/docs/references/eval-metric__supported-metrics
